src/test/ui/issue-5239-1.stderr

```diff
@@ -5,6 +5,8 @@ LL |     let x = |ref x: isize| { x += 1; };
    |                              -^^^^^
    |                              |
    |                              cannot use `+=` on type `&isize`
+   |
+   = note: an implementation of `std::ops::AddAssign` might be missing for `&isize`
```

Comment:
If you add a check for wether `*actual` could have `op` applied, I would point that out with a `span_suggestion`:

```
 error[E0368]: binary assignment operation `+=` cannot be applied to type `&isize`
   --> $DIR/issue-5239-1.rs:14:30
    |
 LL |     let x = |ref x: isize| { x += 1; };
    |                              -^^^^^
    |                              |
    |                              cannot use `+=` on type `&isize`
    |                              help: `+=` can be used on `isize`, you can dereference `x`: `&x`
```